modules/nodes-descheduler-installing.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-descheduler.adoc
+
+[id="nodes-descheduler-installing_{context}"]
+= Installing the descheduler
+
+// TODO:
```

Comment:
cc @ingvagabund

Reply:
My understanding is you will use the console and few mouse clicks. Without operator hub, one can create an index image and a catalogsource manifest to install it. Though, I don't think it will be a recommended way how to do it.